### ISSUE TITLE
Possibility of C++ port?

### ISSUE THREAD

**Issue by adamski** [hij1nx/level-replicator#21]
I am looking around for solutions to replicate a local LevelDB from C++ to a server.

Wondering if it would be feasible to port level-replicator to C++?

**Comment by heapwolf** [comment]
Yes, def. This (well, something similar, has been at the top of my list recently)
